### PR TITLE
Many corrections of the PowerPoint quiz

### PR DESCRIPTION
**Pull request by schoppfe**
- Replacement of " = " by checkboxes " - [x] "
- Adaptation of various punctuation marks according to grammar
- Some correction of upper and lower case letters
- Q2: It's NOT the "Layout tools". In MS PP 2019 it's called "Drawing Tools". There is no such answer but in there is a function called "Arrange"
- Q5: Sould be in the printer window not in the options menu
- Q6: What!? The best tab for adding objects is of course Insert and not Files
- Q8: Grammar (https://support.microsoft.com/en-us/office/rules-for-the-accessibility-checker-651e08f2-0fc3-4e10-aaca-74b4a67101c1?ns=powerpnt&version=16&ui=en-us&rs=en-us&ad=us)
- Q9: No, atleast not in MSPP2019. There you have to use Table Tools -> Design Tab -> Table Styles
- Q12: No no no, Slide Show Tab -> Set Up Slide Show -> Show options -> Loop continuously until 'Esc'
- Q14: "Highlight the content you want to use as a slide title, and select Heading 1 in Styles. Highlight the content you want to use as the slide text, and select Heading 2 in Styles." If your Word document contains no Heading 1 or Heading 2 styles, PowerPoint will create a slide for each paragraph in your content.
- Q15: Corrected from "by date" to "by author
- Q16: No it's automatic. If hidden is selected the courser stays hidden no matter what!
- Q19: Selected an answer
- Q21: Selected an answer
- Added Q22, Q23, Q24, Q25, Q26, Q27, Q28, Q29, Q30, Q31, Q32, Q33

### CODE REVIEW

File: microsoft-power-point/microsoft-power-point-quiz.md [hunk 1]
```diff
@@ -1,156 +1,148 @@
...
 
-#### Q2. when do you use to align objects on a slide to one another?
+#### Q2. What do you use to align objects on a slide to one another?
 
-    = Layout tools
-    Arrange rools
-    Design Ideas feature
-    Picture tools
+- [ ] Layout tools
+- [x] Arrange tools
```
Comment (Ebazhanov) [inline microsoft-power-point/microsoft-power-point-quiz.md:13]:
```suggestion
- [x] Arrange tools // It's NOT the "Layout tools". In MS PP 2019 it's called "Drawing Tools". There is no such answer but there is a function called "Arrange"

```

File: microsoft-power-point/microsoft-power-point-quiz.md [hunk 2]
```diff
@@ -1,156 +1,148 @@
...
 
 #### Q5. Where can you configure advanced PDF options?
 
-    = the Options tab
-    the Home tab
-    the Print dialog box
-    the Share tab
+- [ ] the Options tab
+- [ ] the Home tab
+- [x] the Print dialog box
```
Comment (Ebazhanov) [inline microsoft-power-point/microsoft-power-point-quiz.md:35]:
```suggestion
- [x] the Print dialog box // Should be in the printer window not in the options menu
```

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
Hello, Felix @schoppfe! Thank you for the detailed PR message and I think it would be great if you will move some of your explanations inside the *.md file because after merge they will disappear. I will add some examples please follow them for the rest of your changes. Thank you in advance!

File: microsoft-power-point/microsoft-power-point-quiz.md [hunk 3]
```diff
@@ -161,7 +153,96 @@
...
+- [x] To remove some of the beginning and/or end of the video.
+
+#### Q32. How would you save your layouts, colors, fonts, effects, background styles and content for others to use with their presentations?
+
+- [x] Export the file as a template.
+- [ ] Save the file as a PDF.
+- [ ] Save the file as a POTX.
+- [ ] Save the file as a PPTX.
+
+```Personal Hint: There are multiple file suffixes for templates (.potx, .potm, .pot).```
```
Comment (Ebazhanov) [inline microsoft-power-point/microsoft-power-point-quiz.md:242]: good point 👍

File: microsoft-power-point/microsoft-power-point-quiz.md [hunk 4]
```diff
@@ -161,7 +153,96 @@
...
+
+- [ ] Compress the picture and use the Print (220 ppi) resolution setting.
+- [ ] Select Best scale for slide show, increase the picture resolution and resize the picture to the original size
+- [x] Remove picture compression.
+- [ ] Change the printer property settings to print in a higher resolution.
+
+```Personal Hint:
+File Tab -> Options -> Advanced -> Image Size and Quality -> Do not compress images in file
+File Tab -> Options -> Advanced -> Print -> High quality
+```
```
Comment (Ebazhanov) [inline microsoft-power-point/microsoft-power-point-quiz.md:171]: like

File: microsoft-power-point/microsoft-power-point-quiz.md [hunk 5]
```diff
@@ -1,156 +1,148 @@
...
 #### Q8. What is NOT checked when you run the Accessibility Checker?
 
-    reading order
-    = slide titles
-    missing alt text
-    grammar
+- [ ] reading order
+- [ ] slide titles
+- [ ] missing alt text
+- [x] grammar
```
Comment (schoppfe) [inline microsoft-power-point/microsoft-power-point-quiz.md:57]: - [x] grammar // [Source](https://support.microsoft.com/en-us/office/rules-for-the-accessibility-checker-651e08f2-0fc3-4e10-aaca-74b4a67101c1?ns=powerpnt&version=16&ui=en-us&rs=en-us&ad=us)

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
ah, mist! @schoppfe can you please fix conflicts between your branch and master? Thank you in advance! 
image

**Review by Ebazhanov — APPROVED** [review]
Super, cool! Danke @schoppfe! :punch:
Und einen schönen :sunny: Tag noch

**Pull request merged**